src/editor/contentState/updateCtrl.js

```diff
@@ -1,9 +1,19 @@
+import { ipcRenderer } from 'electron'
```

Comment:
Do not use electron API in `editor` folder, Because codes in `editor` folder will be extracted to new library which can be use for web development, not just electron APP.

Put all preferences in `render/store` is a good practice.

Reply:
I'll take a look at that later :+1: